Orange/modelling/base.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, preprocessors=None, **kwargs):
         self.kwargs = kwargs
         # Make sure to pass preprocessor params to individual learners
         self.kwargs['preprocessors'] = preprocessors
-        self.problem_type = None
+        self.problem_type = self.CLASSIFICATION
```

Comment:
Why is this necessary? Haven't we discussed that it's better to leave this undefined until you get the data? (Apologies if we decided otherwise and I forgot it.)